tests/performance-test/grafana/perftest-dashboard.json

```diff
@@ -2454,9 +2465,10 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "pod_name:container_cpu_usage:sum{$all_stf_pods}",
+          "expr": "pod:container_cpu_usage:sum{$all_stf_pods}",
```

Comment:
So it turns out that on my HAA environment with OCP 4.2, this still needs to be `pod_name`, whereas on the Quicklab 4.3 it needs to be `pod`. I'm pretty sure it used to be `pod_name` in quicklab too, so this is probably still the right change, but I wanted to at least point this out.

Reply:
interesting that's inconvenient